extensions.json

```
@@ -430,7 +422,9 @@
     "repository": "https://github.com/halcyon-tech/code-for-ibmi"
   },
   "halcyontechltd.git-client-ibmi": {
-    "repository": "https://github.com/halcyon-tech/git-client-ibmi"
+    "repository": "https://github.com/halcyon-tech/git-client-ibmi",
+    "location": ".",
+    "prepublish": "[ $VERSION = '0.1.0' ] && mv LICENCE LICENSE || true"
```

It seems this change does not actually do anything, so I think we can remove it

```suggestion
    "repository": "https://github.com/halcyon-tech/git-client-ibmi",
```

---

It does - previously, this extension wasn't published because ovsx couldn't find an open source license.
I assume renaming the file properly will help, does it?